scripts/build-wasm.js

```diff
@@ -31,19 +32,40 @@ let readme = fs.readFileSync(`${dir}/README.md`, 'utf8');
 readme = readme.replace('# ⚡️ Lightning CSS', '# ⚡️ lightningcss-wasm');
 fs.writeFileSync(`${dir}/wasm/README.md`, readme);
 
+const cjsBuild = {
+  entryPoints: [
+    `${dir}/wasm/wasm-node.mjs`,
+    `${dir}/wasm/index.mjs`,
+  ],
+  bundle: true,
```

Comment:
I think we should exclude node_modules here so that napi-wasm doesn't get bundled.